pkg/project/project.go

```diff
@@ -34,6 +36,60 @@ func Create(client *occlient.Client, projectName string) error {
 	return nil
 }
 
+// Delete deletes the project with name projectName and sets the project with name currentProject as current project and returns errors if any
+func Delete(client *occlient.Client, projectName string) error {
+	currentProject := GetCurrent(client)
+
+	projects, err := List(client)
+	if err != nil {
+		return errors.Wrapf(err, "unable to fetch list of projects")
+	}
+
+	//Iterate the project list and see the expected change post deletion
+	for ind, prj := range projects {
+		if prj.Name == projectName {
+			projects = append(projects[:ind], projects[ind+1:]...)
+		}
+	}
+
+	// If there will be any projects post the current deletion,
+	// Randomly choose a project from remainder of project list to set as current
+	if len(projects) > 0 {
+		currentProject = projects[0].Name
+	} else {
+		// Set the current project to NOPROJECT
+		currentProject = "NOPROJECT"
```

Comment:
Shouldn't this be blank / just "" like how it currently is implemented with project?

Reply:
Ack